src/routes/NavigationWrapper.tsx

```diff
@@ -40,16 +70,20 @@ const NavigationWrapper = () => {
   const renderProviderRoutes = () => {
     return (
       <Switch>
-        <Route path={"/helpers"} exact component={Helpers} />
-        <Route path={"/profile"} component={Profile} />
+        <Route
+          path={"/helpers"}
+          exact
+          component={userIsAuthenticated(Helpers)}
+        />
+        <Route path={"/profile"} component={userIsAuthenticated(Profile)} />
       </Switch>
     );
   };
 
   return (
     <>
       {helper ? renderHelperRoutes() : renderProviderRoutes()}
```

Comment:
Kann man im Grunde wie eigenen Components behandeln. Einfach `<HelperRoutes />` und `<ProviderRoutes />`. Könnte dann ebenfalls ausgelagert werden. Was denkst Du?

Reply:
Fände ich ganz schön klein gestückelt dann, so finde ich es vom aktuellen Umfang in Funktionen direkt in der File ziemlich übersichtlich

Reply:
Alles klar, ist mir Recht.